### PR TITLE
fix(install): discovered log files needs to be a string to work within our recipe

### PR DESCRIPTION
**Pull request by sanderblue**
Related: [newrelic/open-install-library #293](https://github.com/newrelic/open-install-library/pull/293)

### CODE REVIEW

**Review by ctrombley — APPROVED** [review]

**Review by zlesnr — DISMISSED** [review]

File: internal/install/recipe_installer_guided.go [hunk 1]
```diff
@@ -153,12 +154,17 @@ func (i *RecipeInstaller) installLogging(ctx context.Context, m *types.Discovery
...
-	// The struct to approximate the logging configuration file of the Infra Agent.
-	type loggingConfig struct {
-		Logs []types.LogMatch `yaml:"logs"`
+	discoveredLogFiles := []string{}
+	for _, logMatch := range acceptedLogMatches {
+		discoveredLogFiles = append(discoveredLogFiles, logMatch.File)
 	}
 
-	r.AddVar("DISCOVERED_LOG_FILES", loggingConfig{Logs: acceptedLogMatches})
+	discoveredLogFilesString := strings.Join(discoveredLogFiles, ",")
```
Comment (Julien4218) [inline internal/install/recipe_installer_guided.go:162]: It's tricky to get this deployed with the library changes.
Could we add a new data element maybe named "NR_DISCOVERED_LOG_FILES" in addition of the existing "DISCOVERED_LOG_FILES", and then in a subsequent release remove the old element "DISCOVERED_LOG_FILES" (after all recipes have been updated to use the new data element).

**Review by Julien4218 — COMMENTED** [review]
@sanderblue I like the change, we'll have a better time with the recipe dealing with a CSV list of filepath than a yaml structure.

However, can we phase the CLI change into 2 releases, adding a new data element while preserving the existing "DISCOVERED_LOG_FILES", and then doing another release later to remove the old data element?

File: internal/install/recipe_installer_guided.go (hunk 1 above)
Comment (sanderblue) [reply internal/install/recipe_installer_guided.go:162]: Oh right, thanks for catching that.

Comment (sanderblue) [reply internal/install/recipe_installer_guided.go:162]: Updated 😎

**Review by zlesnr — APPROVED** [review]

**Review by Julien4218 — APPROVED** [review]
Looks good

**Pull request merged**